.github/workflows/ci.yaml

```diff
@@ -70,7 +70,11 @@ jobs:
           COVERALLS_REPO_TOKEN: ${{ secrets.COVERALLS_REPO_TOKEN }}
 
   deploy-staging:
-    if: contains(needs.initialize.outputs.teams, 'Writers') && (github.ref != 'refs/heads/main')
+    if: >
+      github.ref != 'refs/heads/main' &&
+      (contains(needs.initialize.outputs.teams, 'Writers') ||
+       github.actor == 'dependabot[bot]' ||
```

Comment:
Opted to use `github.actor` rather than `github.triggering_actor`. 

The reason? If one of us manually re-trigger the workflow, the output of `garden github-teams` will contain `Writers` and the job will run.